app/rpc/namespaces/eth/api.go

```diff
@@ -647,7 +647,7 @@ func (api *PublicEthereumAPI) EstimateGas(args rpctypes.CallArgs) (hexutil.Uint6
 
 	// TODO: change 1000 buffer for more accurate buffer (eg: SDK's gasAdjusted)
 	estimatedGas := simResponse.GasInfo.GasUsed
-	gas := estimatedGas + 1000
+	gas := estimatedGas * 12 / 10
```

Comment:
increase 20% gas estimation looks too large to deploy contract
some huge contract may need over 1 million gas

Reply:
gas refund will return unused gas, that's ok